### PR TITLE
VACMS-8359 Home page iteration - blog promo

### PR DESCRIPTION
**Pull request by allisonlu**
## Description

Built out a Promo component for the VAntage Point blog and other news for the Home page. This is part of the Home page redesign iteration for better usability. The component contains an image (has alt text), h3 header, blurb paragraph, and two CTA links. 

See design file:
https://www.sketch.com/s/3aa40506-4be2-46cc-876b-93f1a9f3a857/a/GmRqkmw#Inspector

See ticket: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8359

Note: this is **hard-coded** for now. We will populate it from a data source in another ticket and PR. 

## Testing done
- [x] locally
- [ ] pending user testing

### Mobile
Screen Shot 2022-05-04 at 6 00 57 PM

### Tablet 
Screen Shot 2022-05-04 at 6 01 16 PM

### Desktop
Screen Shot 2022-05-04 at 6 02 01 PM

## Acceptance criteria
- [ ] image has correct aspect ratio (3:4)
- [ ] blurb paragraph wraps fine on all screen widths
- [ ] uses CTA component from VA design system

## Definition of done
- [ ] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/includes/blog-promo.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            <h2 class="vads-u-font-family--sans">Updates on Veteran homelessness</h2>
+
+            <p class="vads-u-padding-right--2 small-desktop-screen:vads-u-padding-right--0">Results of 2021 PIT Count, the annual effort to estimate the number of Americans, including Veterans, without permanent housing.</p>
+
+            <div>
+              <a class="vads-c-action-link--white" href="http://www.va.gov/homeless/">Get housing assistance now</a>
+            </div>
+
+            <!-- this class does not exist in the VA design system... but could? -->
+            <div class="va-h-ruled--white"></div>
```
Comment (timwright12) [inline src/site/includes/blog-promo.html:29]: See other PR for comments about this https://github.com/department-of-veterans-affairs/vets-website/pull/20973/files#diff-ec9531381a0001130bfc443755cd5c42ce6b7daeb0f3e87188d6dddcaea83148R18

**Comment by kind-claudio** [comment]
@allisonlu Noticing how the CTA text is wrapping and falls under the circled arrow icon - the padding looks off from the design. And this also poses the question if the text should wrap beneath the arrow icon or beneath itself, the text which is wrapping, if that makes sense.

**Comment by allisonlu** [comment]
Thanks @kind-claudio, I'm grabbing the [CTA component](https://design.va.gov/components/action-link) straight from the design system without any modifications. For this iteration of the home page, we're keeping it as simple as possible and refining pixels at a later time.

**Review by ryguyk — APPROVED** [review]
Great work!

**Review by timwright12 — APPROVED** [review]

**Pull request merged**